.travis.yml

```diff
@@ -171,17 +195,17 @@ script:
   # build & run tests, build benchmarks
   - ${CABAL} v2-build $WITHCOMPILER ${TEST} ${BENCH} all | color_cabal_output
   # Testing...
-  - ${CABAL} v2-test $WITHCOMPILER ${TEST} ${BENCH} all | color_cabal_output
+  - if ! $GHCJS ; then ${CABAL} v2-test $WITHCOMPILER ${TEST} ${BENCH} all | color_cabal_output ; fi
```

Comment:
Why is `v2-test` not enabled at all for GHCJS? I do realize that three of the test suites have to be disabled due to uninstallable dependencies, but the fourth test suite (`templates`) should still be runnable, right?

Reply:
https://github.com/haskell/cabal/issues/6175

Even one can do something like `find dist-newstyle -name all.js` to find all executable-likes built, I haven't found (yet?) a way to pickup exactly the test-suites from there.